src/devtools/client/debugger/src/components/App.tsx

```diff
@@ -34,6 +34,11 @@ type DebuggerProps = PropsFromRedux & { wrapper: HTMLDivElement };
 class Debugger extends Component<DebuggerProps> {
   shortcuts = new KeyShortcuts({ window, target: this.props.wrapper });
 
+  static childContextTypes = {
```

Comment:
:hand: Just add `static` in front of the other `childContextTypes` field below, which is what I was _trying_ to do when I moved that inside the class.  (or move it back out and do `Debugger.childContextTypes = ` after the definition, which is how I was going to fix this myself.)

Reply:
Oh fuck. I ... missed the other one 😮‍💨 It's been a day